view/categories.js

```diff
@@ -0,0 +1,21 @@
+module.exports = function( vs ){
+
+  // validate required params
+  if( !vs.isset('categories:field') ||
+    !vs.isset('input:categories') ){
+    return null;
+  }
+
+  // base view
+  var view = { terms: {} };
+
+  var value = vs.var('input:categories').get();
+  if (!(value instanceof Array)) {
+    value = [ value ];
+  }
+
+  // terms query
+  view.terms[ vs.var('categories:field') ] = value;
```

Comment:
I'm not exactly sure how to apply that here, so I'm happy to not change it at this time. Thanks for calling it out, as I kinda forgot about that thing.